controllers/comments-controller.js

```diff
@@ -0,0 +1,20 @@
+const { selectComments } = require("../models/comments-model")
+
+function getComments(req, res, next) {
+    const article_id = req.params.article_id;
```

Comment:
You will need to check the reason for the query returning empty rows:
- article exists but has no comments.
- article does not exist.
You will need to check whether the article exists in order to determine this.
I recommend the use of Promise.all() here for efficiency.